platform/core/src/studies/services/qido/studies.js

```diff
@@ -118,9 +118,13 @@ export default function Studies(server, filter) {
   };
 
   const dicomWeb = new api.DICOMwebClient(config);
+  let serverSupportsQIDOIncludeField = server.qidoSupportsIncludeField;
```

Comment:
```suggestion
 server.qidoSupportsIncludeField = server.qidoSupportsIncludeField === undefined ? true : server.qidoSupportsIncludeField;
```

If possible, I think we can avoid creating a new var, and just update value if it's undefined.

Reply:
Looks like GitHub doesn't support multi-line suggestions. Replacing all changes with something like:

```js
server.qidoSupportsIncludeField = server.qidoSupportsIncludeField === undefined 
    ? true
    : server.qidoSupportsIncludeField;
```

Reply:
Added your suggestions to https://github.com/OHIF/Viewers/pull/801